playbooks/roles/ses-rook/tasks/main.yml

```diff
@@ -0,0 +1,104 @@
+---
+- name: Ensure kubectl is properly configured
```

Comment:
Can we call this outside of the `ses-rook` role? To not depend on the `common-deployer` role?

Reply:
We could, but what do we gain here? Again, this role is integrated on socok8s, it only works there. Dow e want to make a generic ses-rook role that its not used anywhere but in this repo and maintain the extra changes that would require a more abstract role?

Reply:
I would say it's more clear (instead of more abstract). If you google for `ansible role` you get things like:
```
Ansible role is an independent component which allows reuse of common configuration steps.
```

If we don't keep roles independent, why are we using them? We could just have one role (eg `socok8s` and have 100 task files.

Reply:
I do not understand, are you arguing against shared code in principle?

Reply:
I do agree with that but no other role in our repo is independent. Its more like a separation of concerns and organization rather than a real role (flashbacks of crowbar all around :fearful: )

See above answer about this.

Reply:
@jsuchome no. I'm arguing for **reusable and shareable** code. I would like to have roles that are independent from each other so the roles could be reused (or even uploaded to ansible galaxy). And the connection between roles is done in the playbooks where the roles are used (instead of including our `common-vars` into every role).
I know that's not what we've done in the past but we could start with it for new roles.